server/src/test/java/org/elasticsearch/usage/UsageServiceTests.java

```diff
@@ -41,6 +42,45 @@
 
 public class UsageServiceTests extends ESTestCase {
 
+    public void testNullHandler() {
+        final UsageService service = new UsageService();
+        expectThrows(NullPointerException.class, () -> service.addRestHandler(null));
+    }
+
+    public void testAHandlerWithNoName() {
+        final UsageService service = new UsageService();
+        final BaseRestHandler horse = new MockRestHandler(null);
```

Comment:
Quality variable name! 👍🐴

Reply:
Little slice of American pie right there. 😉